lab-shawn/README.md

```diff
@@ -0,0 +1,36 @@
+# **Vanilla HTTP RESTful API**
+
+## **Overview**
+
+  This application is a HTTP RESTful API written in vanilla JavaScript. It utilizes the GET, POST, & DELETE methods to fetch, add, and remove files in a file system.
+
+## **How To Use API**
+  * Clone this repository
+  * Open a terminal and run `npm i` to install all the application dependencies
+
+    #### Dependencies
+    * bluebird 3.4.6
+    * node-uuid 1.4.7
+
+    #### Dev dependencies
+    * chai 3.5.0
+    * gulp 3.9.1
+    * gulp-eslint 3.0.1
+    * gulp-mocha 3.0.1
+    * mocha 3.2.0
+    * superagent 3.3.0
```

Comment:
Not necessary to include dependencies in the README since they are accessible in your package.json :)